### DISCUSSION TITLE
Text Classification of documents in business categories

### DISCUSSION THREAD

**Discussion by agonzalezm** [explosion/spaCy#11656] [Help: Other Questions] [answered]
It is possible to use spacy to classify text/documents in categories like if the text is related to: Product, Finance, Legal, HR, Sales, Contracts, Marketing, etc 

Is there any existing models for that? or any other solution that can do this?

**Comment by polm** [comment, marked as answer]
We don't provide a pretrained model for this, no. Our recommendation is that you build your own set of training data, built to your specifications and definitions of categories, and use that to train a model.

I believe there are commercial APIs that provide services like this, but because you're stuck with their definitions you'll spend time figuring out where your definitions don't match up, and troubleshooting will be difficult.

You can also use so-called "zero shot" classification methods, where you just give a set of labels and classify items into them, but again you don't have much control compared to building your own set of examples.
